PiT 23.2: compilation error because resolution failed

### Description

Running the project with 23.2 there is a resolution error that fails compilation
```
[ERROR] Resolution failed. Capabilities satisfying the following requirements could not be found:
    [<<INITIAL>>]
      ⇒ osgi.identity: (&(osgi.identity=com.vaadin.flow.data)(version>=6.0.0))
          ⇒ [com.vaadin.flow.data version=23.2.0]
              ⇒ osgi.wiring.package: (&(osgi.wiring.package=com.vaadin.flow.component)(version>=23.2.0)(version<=23.2.0))
                  ⇒ [com.vaadin.flow.server version=23.2.0]
                      ⇒ osgi.wiring.package: (&(osgi.wiring.package=com.vaadin.pro.licensechecker)(version>=1.5.0)(!(version>=2.0.0)))
    [org.eclipse.jetty.alpn.server version=9.4.35.v20201120]
      ⇒ osgi.wiring.package: (&(osgi.wiring.package=org.eclipse.jetty.alpn)(version>=1.1.3))
```

### How to reproduce

```
git clone https://github.com/vaadin/base-starter-flow-osgi.git 
cd base-starter-flow-osgi
mvn -B -q versions:set-property -Dproperty=vaadin.version -DnewVersion=23.2.0.alpha3
mvn clean install
```

--- a/app/app.bndrun
+++ b/app/app.bndrun
@@ -10,5 +10,5 @@
 	bnd.identity;version='6.0';id='com.vaadin.flow.html.components',\
 	bnd.identity;version='6.0';id='com.vaadin.flow.push',\
 	bnd.identity;version='6.0';id='com.vaadin.flow.osgi',\
-	bnd.identity;version='1.4.1';id='com.vaadin.license-checker'
+	bnd.identity;version='1.6.0';id='com.vaadin.license-checker'
 

--- a/app/pom.xml
+++ b/app/pom.xml
@@ -29,7 +29,7 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>license-checker</artifactId>
-            <version>1.4.1</version>
+            <version>1.6.0</version>
         </dependency>
 
         <dependency>

